src/Bowling.js

```diff
@@ -0,0 +1,121 @@
+var Game = function() {
+  this.score = 0;
+  this.frame = 0;
+  this.frameBall = 1;
+  this.frameScore = 0;
+  this.scoreCard = [];
+  this.pinsInPlay = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10];
+  this.strikeBonusA = 0;
+  this.strikeBonusB = 0;
+  this.spareBonusA = 0;
+  this.spareBonusB = 0;
+  this.knockedDown = [];
+};
+
+Game.prototype.play = function() {
+  if(this.frame < 10) {
+    this.knockedDown = [];
+    this.frameBall = 0;
+    this.rollBall();
+    this.bonusPoints();
+    this.strikeBonusA-=1;
+    this.strikeBonusB-=1;
+    this.spareBonusA-=1;
+    this.spareBonusB-=1;
+    this.scoreCard.push(this.knockedDown);
+    this.frame += 1;
+    if(this.frame==10){
+    this.eleventhFrame();
```

Comment:
So many variables is a code smell - consider ways to extract this into functions that calculate 'on-the-fly' the current state of the object.